### ISSUE TITLE
difference between VAEG-BIN and GraphMB?

### ISSUE THREAD

**Issue by taylorreiter** [AndreLamurias/GraphMB#14]
Hello! I recently came across GraphMB and VAEG-BIN, and was curious what the differences are between the two approaches or if they have been combined into the same project. If they have not been combined, is the code available for VAEG-BIN? Thank you!

**Comment by AndreLamurias** [comment]
Hi Taylor, thanks for the interest. The VAEG-Bin paper generalizes what we did with GraphMB to more types of Graph Neural Networks and adds a loss component based on SCG. From a technical point of view it's also implemented in tensorflow instead of pytorch. We are integrating those changes into this repo, but it's still a work in progress. If you want you can check the dev and gat branches, but do it at your own risk, since it might have bugs and other issues.

**Comment by jianshu93** [comment]
Hello Andre,

I downloaded the dev branch but it seems python setup install did not generate the graphmb binary and put it in path. Is there something that I missed.

Thanks,

Jianshu

**Comment by AndreLamurias** [comment]
Hi Jianshu, as I said, that code is not fully working yet, I cannot fix your issues, do it at your own risk.

**Comment by taylorreiter** [comment]
Thank you @AndreLamurias! This was super helpful and I've been able to point a few groups who had the same question at your answer 🎉
